sql/planbuilder/ddl.go

```diff
@@ -1020,9 +1020,6 @@ func (b *Builder) tableSpecToSchema(inScope, outScope *scope, db sql.Database, t
 		defaults[i] = cd.Type.Default
 		generated[i] = cd.Type.GeneratedExpr
 
-		cd.Type.Default = nil
-		cd.Type.GeneratedExpr = nil
-
```

Comment:
I'm sure why we set these to nil, but this was causing issue where column default value gets lost when the query goes through prepare and gets cached, so when we get the query from the cache to execute, the column default value is nil.